### ISSUE TITLE
log only certain levels

### ISSUE THREAD

**Issue by Paul-Todd** [jorgecoca/lumberdash#46]
Hi,

Great library! Especially now supporting multiple loggers.

However I have a feature suggesting in that I would like to be able to only log certain levels of errors.
Ideally this level would be per logger but I guess the least impact would be in the main Lumberdash client.

Usecase:
Log error and fatal's to Crashlytics and the rest to the console.
Thanks

Paul

**Comment by jorgecoca** [comment]
@Paul-Todd I believe you could do that with the existing API, since `lumberdash` supports an `exceptFor` field: https://github.com/bmw-tech/lumberdash/blob/4c30df86afbb6ab1a3f883b86886cd56b4891e2d/lib/src/lumberdash.dart#L50

Something like this (pseudo-code):

```dart
final shouldFilter = error.runtimeType == NonFatalError;
final exceptForClients = shouldFilter ? [PrintLumberdash] : [];
logError(error, exceptForClients);
```

If you have an idea for a different API, please share it with us and we can take a look ;)

**Comment by IchordeDionysos** [comment]
Yes, this is possible that way, but not very convenient don't you think?

Suppose you put `logMessage` all over in your project and then you add another logger for only `logError` then one would have to change every single `logMessage` to also include the new logger.

Having this option for some logs is cool, but the more common use-case and far more useful use-case would be to restrict it at the logger level.

You might say now: 
Well, you can write your own logger. Yes, this works.

But it's not possible to customize an existing logger.

For example: We only want to log errors to sentry, so doing something like this:

```dart
SentryLumberdash.withDsn(
  dsn: "...", 
  logLevels: [LogLevel.warning, LogLevel.error]
);
```

The example above would give the developer the most freedom.

**Comment by fwagner** [comment]
We definitly see value in introducing the concept of log levels to lumberdash and being able to configure the routing of messages to appenders in a central location. It's on our roadmap and we may be able to add this later this year.
